server.js

```diff
@@ -1,18 +1,22 @@
 require("dotenv").config();
 /* ==== External Modules ==== */
 const express = require("express");
-//const morgan = require('morgan');
+const morgan = require('morgan');
 const session = require("express-session");
 const passport = require("passport");
-const indexRoutes = require('./routes/index');
-const userRoutes = require('./routes/user');
+const router = express.Router();
 
 
 
-//const methodOverride = require("method-override");
 
-/* ==== Internal Modules ==== */
 
+/* ==== Internal Modules ==== */
+const indexRoutes = require('./routes/index');
+const userRoutes = require('./routes/user');
+const excerRoute = require('./routes/workoutlog'); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const excerRoute = require("./routes/workoutlog"); 
```